README.md

```diff
@@ -71,7 +71,7 @@ export LLVM_DIR=<installation/dir/of/clang/11>
 export CLANG_TUTOR_DIR=<source/dir/clang/tutor>
 mkdir build
 cd build
-cmake -DCT_LLVM_INSTALL_DIR=$LLVM_DIR $CLANG_TUTOR_DIR/HelloWorld/
+cmake -DCT_LLVM_INSTALL_DIR=$LLVM_DIR -DCMAKE_EXPORT_COMPILE_COMMANDS=ON $CLANG_TUTOR_DIR/HelloWorld/
```

Comment:
Hm, this shouldn't be needed - why would we need this here?

Reply:
Sorry, It is not needed here.